provisioning/transport/amqp/src/Utils.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.Devices.Provisioning.Client.Transport.Amqp
+{
+    static class Utils
```

Comment:
Used only by the ClientWebSocket - we should switch to ArraySegment and remove this class.